node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -408,9 +410,24 @@ removeNextUpdate = wsReadyUpdates %= drop 1
 testReset :: Update ()
 testReset = put def
 
-updateHistoryCache :: CId Wal -> [TxHistoryEntry] -> Update ()
-updateHistoryCache cWalId cTxs =
-    wsHistoryCache . at cWalId ?= cTxs
+data HistoryCacheUpdate
+    = InitHistoryCache
+    | InsertToHistoryCache [TxHistoryEntry]
+    | RemoveFromHistoryCache [TxHistoryEntry]
+
+-- | Modifies cached history, returns whether call succeeded.
```

Comment:
"returns whether call succeeded" eh? It returns `()`.

Reply:
Oh thanks, outdated info